mysql/const.go

```diff
@@ -58,6 +58,54 @@ const (
 	ServerPSOutParams              uint16 = 0x1000
 )
 
+// ServerStatus2Str convert server status to string.
+// Param state is a bit-field. (e.g. 0x0003 = "in transaction; autocommit").
+func ServerStatus2Str(state uint16) string {
+	// l collect server status strings.
+	var l []string
+	// status is a slice of all defined server status in ascending order.
+	var status []uint16
+	status = append(
```

Comment:
Every time this function is called, it makes an allocation here?
How about moving the definition out so the allocation is avoided